components/cli/pkg/commands/instance/logs.go

```diff
@@ -24,32 +24,32 @@ import (
 	"github.com/cellery-io/sdk/components/cli/cli"
 )
 
-func RunLogs(cli cli.Cli, cellName, componentName string, sysLog bool) error {
+func RunLogs(cli cli.Cli, instanceName string, componentName string, sysLog bool, follow bool) error {
```

Comment:
Can be rewritten as "sysLog, follow bool".

Reply:
Put it just for the sake of readability since there are arguments with three types in this signature